main.py

```diff
@@ -1,10 +1,43 @@
 import init_django_orm  # noqa: F401
-
+import json
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json") as file:
+        players = json.load(file)
+
+        for nickname, person in players.items():
+            race_info = person["race"]
+
+            race, _ = Race.objects.get_or_create(
+                name=race_info["name"],
+                description=race_info["description"]
+            )
+
+            for skill in race_info["skills"]:
+                Skill.objects.get_or_create(
+                    name=skill["name"],
+                    bonus=skill["bonus"],
+                    race=race
+                )
+
+            guild_info = person["guild"]
+            if guild_info:
+                guild, created = Guild.objects.get_or_create(
+                    name=guild_info["name"],
+                    description=guild_info["description"]
+                )
+            else:
+                guild = None
```

Comment:
You can create this viriable at the start and do not use else block .

If your condition is triggered, the contents of the variable will simply be overwritten